docs/geedocs/5.3.0/answer/7160007.html

```diff
@@ -49,6 +49,16 @@ <h5>
       This can be done by configuring the <code>PURGE</code> and <code>PURGE_LEVEL</code> settings in <code>/etc/opt/google/XMLparams</code>. 
       This file must be created manually. View the <a href="../answer/176738.html">Configure Fusion performance</a> document for more details.</p>
     <p><strong>Added a <code>long_version</code> parameter to the SCons build. </strong>This will allow the user to override the long version string.</p>
+
+    <p><strong>Added PUBLISHER_ROOT size checking.</strong>
+       ASSET_ROOT checking is performed in fusion installer - not duplicated in server installer.
+       Added <code>-pgu pguser</code>, <code>-au apacheuser</code>, and <code>-g group</code> options.
+       Added prompt and summary output similar to fusion installer.</p>
+
+    <p><strong>We have improved the performance of large builds</strong> by consolidating some of the notifications that an asset's child or input has changed. By preventing (some) duplicate notifications we reduce the number of times each asset has to calculate its state, significantly improving performance.
```

Comment:
I think this is too low level for release notes. It would read better something like this:

<p><strong>Improved system manager performance for large builds.</strong>
This release eliminates some duplicate processing in system manager, which reduces the amount of time it takes to initiate large builds.